README.md

```diff
@@ -14,23 +14,61 @@ Clone this repo: <a href = "https://github.com/Swarathmica-infraspec/payout-mana
 GO-VERSION: 1.25.0
 
 The project contains payoutmanagementsystem/ <br>
-    - .github/workflows/payoutManagementSystem.yml <br>
-    - payee/ <br>
-        - payee.go <br>
-        - payee_test.go <br>
-    - go.mod <br>
-    - go.sum <br>
-    - README.md <br>
+
+- .github/workflows/payoutManagementSystem.yml <br>
+- payee/
+  - payee.go <br>
+  - payee_test.go <br>
+  - payee_db.sql <br>
+  - payeeDAO.go <br>
+  - payeeDAO_test.go <br>
+- go.mod <br>
+- go.sum <br>
+- README.md <br>
+
 
 NOTE: Only email ids with .com are supported.
 
-# Run Tests
 
-Test can be run by executing the below command in the terminal
-  go test -v ./...
+
+# Database Setup
+
+We use PostgreSQL running inside Docker for persistant storage.
```

Comment:
[documentation-update] mention postgres version in readme
@Swarathmica-infraspec  @KirthiInfra